src/styles/theme.js

```diff
@@ -7,13 +7,14 @@ const theme = {
     deepGrey: '#222222',
     hoverGrey: '#DBDBDB',
     pink: '#FF09D6',
-    skyblue: '#4C83C2',
+    middleBlue: '#4C83C2',
     blue: '#4C5AEE',
     grayScale: ['#eee', '#ddd', '#555', '#888'],
     transparentBlack0: 'rgba(0, 0, 0, 0.05)',
     transparentBlack1: 'rgba(0, 0, 0, 0.1)',
     transparentBlack2: 'rgba(0, 0, 0, 0.2)',
-    transparentBlack3: 'rgba(0, 0, 0, 0.4)',
+    transparentBlack3: 'rgba(0, 0, 0, 0.3)',
+    transparentBlack4: 'rgba(0, 0, 0, 0.4)',
```

Comment:
이부분은 공용 이기때문에 고치려면 조금 힘들것 같은데 우선 의견 올려봅니다. 

```suggestion
transparentBlack: ['rgba(0, 0, 0, 0.05)', 'rgba(0, 0, 0, 0.1)', ....]
```